src/System.Text.Json/tests/Serialization/WriteValueTests.cs

```diff
@@ -25,5 +25,41 @@ public static void CanWriteValueToJsonArray()
             string json = Encoding.UTF8.GetString(memoryStream.ToArray());
             Assert.Equal("{\"test\":[1]}", json);
         }
+
+        public class CustomClassWithEscapedProperty
+        {
+            public int pizza { get; set; }
+            public int hello\u6C49\u5B57 { get; set; }
+            public int normal { get; set; }
+        }
+
+        [Fact]
+        public static void SerializeToWriterRoundTripEscaping()
+        {
+            const string jsonIn = " { \"p\\u0069zza\": 1, \"hello\\u6C49\\u5B57\": 2, \"normal\": 3 }";
```

Comment:
Hmm... GitHub doesn't like this for some reason.
![image](https://user-images.githubusercontent.com/6527137/64226198-4387d300-ce93-11e9-8e4a-faf5cb63e6b5.png)